gnoland/website/static/css/app.css

```diff
@@ -27,49 +66,49 @@ input {
 }
 
 a {
-  color: #25172a;
+  color: var(--link-color, #25172a);
 }
 
 input {
   font-family: "Roboto Mono", "Monaco", monospace;
-  background-color: #eee;
+  background-color: var(--input-background-color, #eee);
   border: 1px solid #ccc;
-  color: black;
+  color: var(--text-color, #000);
   width: 25em;
 }
 
 blockquote {
-  background-color: #ddd;
+  background-color: var(--quote-background, #ddd);
 }
 
 blockquote blockquote {
   margin: 0;
-  background-color: #aaa2;
+  background-color: var(--quote-2-background, #aaa4);
```

Comment:
I'm just curious about the reason behind this change (i.e `#aaa2` -> `#aaa4`)

Rest LGTM 💯

Reply:
Ah - that's an oversight. I had changed it to `#aaa` considering the 2 at the end was a typo, but I added it back later after realising it wasn't. Should be alright though - the opacity should only have changed by a 0.125 :)